### PR TITLE
Fixing Search Results padding/margins, halving animations

### PR DESCRIPTION
**Pull request by TotallyGeeky**
Adjusted values to create more even/consistent spacing for standard search results on the page. 
Adjusted values for title padding to center title + favicon, not just title. Removed unused padding for favicon in sub-results.
Tested live and seems to respond as expected when font/zoom changes. 
Halved animation value/doubled speed on most animations, personally I think it makes it feel snappier without removing them entirely. 

Outstanding issue, still investigating: nav menu item "Lens" on hover is misaligned/miscolored, mis-sized for space.

### CODE REVIEW

**Comment by TotallyGeeky** [comment]
As you can see by my commenting in the wrong place initially, I don't know much about git so please forgive if I did anything dumb/rude.

**Comment by pdanzma** [comment]
Hey, thanks for your effort! You're right, the animation is better at 0.3s and the other changes are awesome! The only thing that bothers me is the padding of the title. If you could remove it, I'd merge it :)

File: custom.css [hunk 1]
```diff
@@ -75,10 +75,11 @@ a.wiki-nw-link:hover{background-color:var(--button-hover-color);color:#1b1b1b}
 .__sri-time{background-color:var(--time-stamp);color:var(--primary-700);user-select:none}
 .__sri-time.--new{background-color:var(--time-stamp);color:var(--primary-700);user-select:none}
 .sri-group,.search-result,.scene{background-color:#1e2028;border-radius:5px;opacity:0;transform:translateY(20px);animation:fadeInUp .6s ease-out forwards}
+._0_TITLE{padding-left: 1.5rem}
```
Comment (pdanzma) [inline custom.css]: Please remove this line. Then we are ready to go :)

**Comment by TotallyGeeky** [comment]
You got it, boss!

**Comment by pdanzma** [comment]
Thanks a lot my man :D

**Pull request merged**